### PR TITLE
Add decision making process

### PR DESCRIPTION
**Pull request by pavithraes**
## Reference Issues or PRs

#158 

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [x] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: docs/docs/community/decision-making.md
Comment (kcpevey) [inline docs/docs/community/decision-making.md:26]:
```suggestion
## Major updates - Request-for-discussion (RFD) process
```

Comment (kcpevey) [reply docs/docs/community/decision-making.md:26]: Just to make this section header and the next header have the same format.

**Review by kcpevey — APPROVED** [review]
LGTM. I just had a few minor adjustments to request.

File: docs/docs/community/decision-making.md
Comment (pavithraes) [reply docs/docs/community/decision-making.md:13]: I think that's currently being handled by the maintainers team and on an ad-hoc basis? I think we can clarify the release timeline in https://www.nebari.dev/docs/community/maintainers/release-process-branching-strategy (which needs an update as well)

**Comment by pavithraes** [comment]
Thanks for the review, will merge once CI passes :)

**Pull request merged**
